docker-compose.yml

```diff
@@ -82,14 +79,14 @@ services:
       - "${MAP_HOST_PULSAR:-127.0.0.1}:6650:6650"
       - "${MAP_HOST_PULSAR:-127.0.0.1}:8081:8080"
     environment:
-      PULSAR_MEM: " -Xms512m -Xmx512m -XX:MaxDirectMemorySize=2g"
+      PULSAR_MEM: " -Xmx256M"
```

Comment:
I would keep `-Xms512m` to avoid pauses for memory allocations. It's one of the best practices to specify both -Xms and -Xmx and set them to the same value.

Reply:
So you mean "-Xms256M -Xmx256M" or "-Xms512M -Xmx512M"? I did multiple runs with 256 and that did not cause any issue.

Reply:
Yes, setting them to the same value removes the need for heap resizes, which is expensive and tricky operation.